ext/redhat/puppet.conf

```diff
@@ -0,0 +1 @@
+puppet.conf
```

Comment:
So for the master, an empty puppet.conf won't work currently. I ran into this with @rlinehan. The puppet master process will run as the puppet user to reduce privilege, which means that the defaults it will see are ~/.puppet/var and friends.
I think it probably makes sense to set vardir, logdir and rundir in puppet.conf for the master run mode. This seems less than ideal because really the master shouldn't need config overriding to run properly, but the only alternative I can think of is to update the which_dir method to do something special if running in the master run mode and the puppet user (as masters will be), which seems weird as well.